trie.go

```diff
@@ -20,8 +20,8 @@ type TrieNode struct {
 	// Some thought was put into alignment here. The following three fields fit
 	// within an 8-byte required alignment on x86_64 at least
 	size     uint32
-	height   uint16
```

Comment:
I had to change the names of these elements because when I changed the names of the methods, they clashed.